src/helperFunctions/compare_sets.py

```diff
@@ -10,14 +19,37 @@ def remove_duplicates_from_unhashable(unhashable_list: List[T]) -> List[T]:
 
 
 def remove_duplicates_from_list(list_):
+    '''
```

Comment:
Actually why do we have two of these? We should really only have this function and call the unhashable variant on an error regarding unhashability ...

Reply:
I agree -> changed